src/searcher.rs

```diff
@@ -139,6 +157,7 @@ impl Searcher {
         count: bool,
         order_by_field: Option<&str>,
         offset: usize,
+        order_by_order: Order,
```

Comment:
How strongly do you feel that this should be called `order_by_order` instead of just `order`?

Reply:
Not particularly strongly here. Just thought it'd be more specific about which other parameter its associated with but I don't anticipate another ordering field any time soon?

Reply:
I can live with `order_by_order`, but it feels "unpythonlike" 😄 .  We can hear from the other maintainers how they feel.

Reply:
To add yet another name to consider, how about `order_in_direction` (and matching `enum Direction`)?

Reply:
I went with `order` (and a matching `enum Order`). I have a strong preference for an API surface and naming convention that closely mimics the original `tantivy` APIs, since I often go to those docs to see how things work in these bindings (as an outside user) :)

Reply:
Same. I wish that `order_by_field` was just called `order_by`, but we can't easily change that now probably.